python/tank/templatekey.py

```diff
@@ -66,6 +76,27 @@ def __init__(self,
         if not all(self.validate(choice) for choice in self.choices):
             raise TankError(self._last_error)
     
+    @property
+    def choices(self):
+        """
+        :returns:    List of choices available for this key
+        """
+        return self._choices.keys()
+    
+    @property
+    def choice_labels(self):
```

Comment:
I wonder if it makes sense to return the values like this, free-floating, detached from the keys?
Could `get_choice_label` and `choice_labels` both be removed in favour of a method that simply returns the entire dictionary?